src/wallet/rpcwallet.cpp

```diff
@@ -2557,10 +2557,10 @@ UniValue listunspent(const UniValue& params, bool fHelp)
             "    \"account\" : \"account\",    (string) DEPRECATED. The associated account, or \"\" for the default account\n"
             "    \"scriptPubKey\" : \"key\",   (string) the script key\n"
             "    \"amount\" : x.xxx,         (numeric) the transaction amount in " + CURRENCY_UNIT + "\n"
-            "    \"confirmations\" : n       (numeric) The number of confirmations\n"
-            "    \"ps_rounds\" : n           (numeric) The number of PS round\n"
+            "    \"confirmations\" : n,      (numeric) The number of confirmations\n"
             "    \"spendable\" : xxx,        (bool) Whether we have the private keys to spend this output\n"
-            "    \"solvable\" : xxx          (bool) Whether we know how to spend this output, ignoring the lack of keys\n"
+            "    \"solvable\" : xxx,         (bool) Whether we know how to spend this output, ignoring the lack of keys\n"
+            "    \"ps_rounds\" : n           (numeric) The number of PS round\n"
```

Comment:
I don't understand why this was moved... and why the actual code wasn't changed to match the new field order here in help.

Reply:
This is to avoid future conflicts for upcoming backports. Especially the ones that reformat all the RPCs help messages. Should have made that clear in the commit.

I added another commit on top that also rearranges the code.

I also have an item on my TODO list to go through all RPCs when we are done with backporting and fix formatting errors introduces by merge conflict resolution. Unfortunately it's not so easy to spot these when looking at the escaped strings or diffs.